src/app/portfolio/page.tsx

```diff
@@ -182,6 +197,7 @@ export default function Portfolio() {
           totalBalance={totalBalance}
           hideLowBalance={hideLowBalance}
           setHideLowBalance={setHideLowBalance}
+          stakingPositions={Object.values(stakingPositions)} // Pass staking positions here
```

Comment:
Not very important, but `stakingPositions` should keep its type as much as possible when it's just moved around.
The `Object.values()` should be done as early or as late as possible (so either right when `stakingPositions` is defined, or when it's used in `AssetsBreakdown`). This is just to avoid confusion of variables with the same name but with types that change along the way.